styx-api-service/src/test/java/com/spotify/styx/api/BackfillResourceTest.java

```diff
@@ -245,7 +245,7 @@ public void shouldFilterBackfillsOnComponentEvenWhenInactive() throws Exception
     storage.storeBackfill(BACKFILL_4.builder().allTriggered(true).build());
 
     final String uri = path(String.format("?showAll=true&component=%s",
-                                          BACKFILL_4.workflowId().componentId()));
+        BACKFILL_4.workflowId().componentId()));
```

Comment:
Please avoid unnecessary whitespace changes in otherwise unchanged code. Makes the PR noisier and harder to review.